### PR TITLE
#166645992 setup docker for backend

### PR DESCRIPTION
**Pull request by ladaposamuel**
#### What does this PR do?
This PR will add docker configuration files and update read me with docker setup instructions.

#### Description of Task to be completed?
- Create` Dockerfile` to assemble the image
- Create the `docker-compose.yml`
- Create the `.gitignore`
- Update `Readme.md` with docker setup instructions

#### How should this be manually tested?
- CLONE this repo
- open the cloned directory
- Install Docker
- Change to the application root directory
- Build a docker image with the following command `docker build -t  storm-backend . `
- Run `docker-compose up`

#### What are the relevant pivotal tracker stories?

[#166645992](https://www.pivotaltracker.com/story/show/167727722)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,10 @@ Barefoot Nomad - Making company travel and accomodation easy and convinient.
...
 Make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.
 
----
\ No newline at end of file
+---
+## Docker Development Setup
+
+- Install Docker
+- Change to application root directory
+- Build a docker image with the following command `docker build -t . storm-backend`
```
Comment (tejiri4) [inline README.md]: change you documentation from `docker build -t . storm-backend` to `docker build -t storm-backend .` . and `docker-composer up` to `docker-compose up`

**Comment by tejiri4** [comment]
Your commit message is not following the laid down format

**Review by tejiri4 — CHANGES_REQUESTED** [review]
Nice work look into the comment I dropped above

File: README.md (hunk 1 above)
Comment (ladaposamuel) [reply README.md]: Fixed

**Comment by ladaposamuel** [comment]
> Your commit message is not following the laid down format

I have updated my commit message, please kindly review.

**Review by Mcdavid95 — APPROVED** [review]
LGTM😁

**Comment by tejiri4** [comment]
Can we make your commit message multi line and follow the defined format specified on slack?

**Comment by ladaposamuel** [comment]
> Can we make your commit message multi-line and follow the defined format specified on slack?

I have updated the commit message using the specified one on slack.

**Comment by tejiri4** [comment]
Add your pivotal tracker ID to the commit message. There is no footer in your commit message sync with @kola-1

**Review by komsic — APPROVED** [review]
Nice work Samuel. Please share the resources you used to achieve this task on Slack

**Review by jsbuddy — APPROVED** [review]
Nice work @ladaposamuel. LGTM

**Comment by ladaposamuel** [comment]
> Add your pivotal tracker ID to the commit message. There is no footer in your commit message sync with @kola-1

Oops... I missed it, Added now.

**Comment by tejiri4** [comment]
Change `Finishes` to `Maintains` in your footer for chore. Do amend your commit message

**Comment by ladaposamuel** [comment]
> Change `Finishes` to `Maintains` in your footer for chore. Do amend your commit message

Fixed.
Please, what is the difference between Finishes and Maintains?

**Review by tejiri4 — APPROVED** [review]
LGTM

**Pull request merged**